### PR TITLE
Some odds and ends, and enable day theme (!)

### PR DESCRIPTION
**Pull request by michaelkirk**
I addressed a few lingering things, and I *believe* we're ready to enable day theme by default (🎉)  and ditch the legacy day theme, so I've done that. Let me know if you'd rather wait.

## Tab legibility

**before**
Screen Shot 2021-02-26 at 3 58 47 PM

**after**
Screen Shot 2021-02-26 at 3 57 31 PM
Screen Shot 2021-02-26 at 3 57 16 PM

## Button hover state in pregame

**restore button hover state and applied theme text color**
Screen Shot 2021-02-26 at 3 41 38 PM

Pregame is kind of a weird beast. The current design isn't really compatible with our new themes, so I haven't tried to make it fit that paradigm. At some point we should redesign it to look more like the rest of the app, but until then I had to hardcode some stuff to keep it looking reasonable.

### CODE REVIEW

File: game/src/challenges/cutscene.rs [hunk 1]
```diff
@@ -221,8 +223,8 @@ fn make_panel(
             .margin_above(100),
             Widget::col(vec![
                 Widget::row(vec![prev.margin_right(40), next]).centered_horiz(),
-                ctx.style()
-                    .btn_outline_text("Skip cutscene")
+                ButtonStyle::outline_dark_fg()
```
Comment (michaelkirk) [inline game/src/challenges/cutscene.rs:226]: hardcoded references to specific button styles in pregame for now 🤷

File: map_gui/src/colors.rs [hunk 2]
```diff
@@ -172,8 +174,19 @@ impl ColorScheme {
         cs
     }
 
+    fn pregame() -> ColorScheme {
```
Comment (michaelkirk) [inline map_gui/src/colors.rs:177]: Secretly this is basically the "legacy day theme". It lives on in pregame only for now.

File: widgetry/src/style/buttons.rs [hunk 3]
```diff
@@ -224,6 +224,55 @@ impl<'a> StyledButtons<'a> for Style {
...
+    }
+
+    pub fn plain_icon_text(&self, image_path: &'a str, text: &'a str) -> ButtonBuilder<'a> {
+        self.plain()
+            .label_text(text)
+            .image_path(image_path)
+            .image_dims(ScreenDims::square(18.0))
+    }
+
+    pub fn outline_text(&self, text: &'a str) -> ButtonBuilder<'a> {
```
Comment (michaelkirk) [inline widgetry/src/style/buttons.rs:256]: For ease of sharing styles with pregame I've slightly rephrased some methods from StyledButtons (which is a trait that EvnetCtx implements) onto ButtonStyle. 

I'm actually going to attempt a refactor based on this to try to get rid of the cartesian explosion of methods on StyledButtons. The result will be less method definitions, but I'm not yet sure if the resultant API will be better. I'll have a go at it and you can let me know what you think.

File: map_gui/src/options.rs [hunk 4]
```diff
@@ -55,7 +55,7 @@ impl Options {
             debug_all_agents: false,
 
             traffic_signal_style: TrafficSignalStyle::BAP,
-            color_scheme: ColorSchemeChoice::DayMode,
+            color_scheme: ColorSchemeChoice::Pregame,
```
Comment (dabreegster) [inline map_gui/src/options.rs:58]: This effectively continues to use the old day mode in the 15min tool, parking mapper, OSM viewer, etc. That seems fine for now. I'll have a look this weekend and cut them over to the new day mode, under the assumption that we're getting rid of the legacy day theme once some of the cutscene stuff is cleared up.

File: widgetry/src/style/buttons.rs (hunk 3 above)
Comment (dabreegster) [reply widgetry/src/style/buttons.rs:256]: Sounds like a promising refactor. The levels of indirection here definitely make it a bit tedious to figure out what's going on.

**Review by dabreegster — APPROVED** [review]
Woohoo, the payoff after all the hard work! I poked around and found a few small issues, but nothing blocking. I can work on fixing some of these this weekend if you're done.

Signal editor background is pretty dark now:
Screenshot from 2021-02-26 16-19-50

Inactive confirm button isn't even legible. Arguably it doesn't need to be since it's inactive, but this seems dramatic.
Screenshot from 2021-02-26 16-18-06

Night mode disabled prev/next suddenly gets a background
Screenshot from 2021-02-26 16-17-03

**Pull request merged**